python/BioSimSpace/Align/_align.py

```diff
@@ -145,9 +154,32 @@ def generateNetwork(molecules, names=None, work_dir=None, plot_network=False,
     if type(plot_network) is not bool:
         raise TypeError("'plot_network' must be of type 'bool'.")
 
+    # Validate the propert map.
     if type(property_map) is not dict:
         raise TypeError("'property_map' must be of type 'dict'")
 
+    # Validate the scores file.
+    if scores_file is not None:
+      if type(scores_file) is not str:
+          raise TypeError("'input_scores' must be of type 'str'")
+      if names is None:
+          raise ValueError("'names' must be defined when passing 'scores_file' to LOMAP.")
+
```

Comment:
It would be good to validate that the file exists before opening it so that we can report a useful message to the user, e.g.:

```python
if _os.path.isfile(scores_file):
    # Do stuff
    ...
else:
    raise IOError(f"The scores file didn't exist: {scores_file}")
```

(The contextmanager would raise an exception for us, but it's nice to make it a little clearer.)

Reply:
see e69de1fc4b19487ce19247ae83369b34d1a3e419